### ISSUE TITLE
images IPTC tags (photo metadata) are not updated when tags are [added/removed]

### ISSUE THREAD

**Issue by nicolastaffin** [LycheeOrg/Lychee#545]
### Detailed description of the problem [REQUIRED]
Lychee 4 imports images with IPTC tags, which is great, but it seems these tags are stripped on export, is it a bug or a feature ? And is it configurable somewhere ?

### Steps to reproduce the issue
**Steps to reproduce the behavior:**
1. import an image which has tags
2. Tags appear on the info panel yay!
3. download the image on your desktop
4. check image meta in Bridge : no tags…

### Browser and system
ffirefox mac

**Comment by ildyria** [comment]
Depends which images you are talking about ?

**The full size one (thus in the `big` folder) should be identical from the one.**

We do strip the data from the medium and smaller sized one.

**Comment by nicolastaffin** [comment]
Thanks to be so fast answering : OK yes if i download using the "original" option i get the original tags still there. But no tag that would have been added in lychee tag editor.

**Comment by ildyria** [comment]
Yes, we do not modify the images. Lychee is more of a online gallery than an image editor / organizer such as Adobe Bride / Lightroom, etc.

We may be interested in a PR if added, but I personally will not be working on such feature.

**Comment by nicolastaffin** [comment]
Hum, you are right, my use case is more a shared photo collection, with albums and tags enrichment for later efficient search and use (download), and yes, i think image metadata tag info updating on export would be important (in my usecase, but also because keeping obsolete tags in image, especialy when you have a tag editor, can be misleading). Not sure i'm skilled to mess with that code, i'd like to.

**Comment by ildyria** [comment]
> Not sure i'm skilled to mess with that code, i'd like to.

It does not hurt to try.
Here is the entry point you will likely be looking for: https://github.com/LycheeOrg/Lychee/blob/master/app/Http/Controllers/PhotoController.php#L291

**Comment by nicolastaffin** [comment]
Thanks, you are teasing me… i'll check that.
